firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+KeyCommands.swift

```diff
@@ -97,11 +86,8 @@ extension BrowserViewController {
                                      object: .keyCommand,
                                      extras: ["action": "go-forward"])
 
-        guard let tab = tabManager.selectedTab, tab.canGoForward else { return }
-
-        if !contentContainer.hasHomepage {
-            tab.goForward()
-        }
+        store.dispatch(GeneralBrowserAction(windowUUID: windowUUID,
```

Comment:
And here: Dispatching the action should only happen if the feature flag for toolbar refactor is turned on. The old code should not be changed.